iSkyLIMS_drylab/templates/iSkyLIMS_drylab/pendingServices.html

```diff
@@ -71,7 +71,7 @@ <h1><div class="color-heading">Pending Services</div></h1>
                                                             <tbody>
                                                                 {% for s_id, serv, user, c_date, res, handle, q_date, e_date  in values %}
                                                                     <tr>
-                                                                            <td><a href="/drylab/displayService={{ s_id }}">{{ serv }}</a> </td>
+                                                                            <td><a href="/drylab/display_service={{ s_id }}">{{ serv }}</a> </td>
```

Comment:
url has to be using camelCase sin taxis.
Revert the change to : 
**displayService**